### ISSUE TITLE
Sorting of variant table on other columns

### ISSUE THREAD

**Issue by molucorner** [Clinical-Genomics/scout#5031]
Currently the list of variants is sorted on the rank, but it would be useful to be able to sort on the other column as well. (And perhaps to be able to sort on gene and position within the gene.) Rank would still be the default.

**Comment by dnil** [comment]
Hi @molucorner! Thank you for the suggestion - as I'm sure you can imagine you are not the first. We strongly try to avoid this for two reasons. One part is computational: each sort index would incur a fairly decent compute cost on the variant collection. We are at some 500M and counting, so the indexes are not free. Another is pedagogical (for us devs): if there is a use case where a variant should systematically be seen before another variant, we would very much like to know it so that the rank model can be adjusted. Tied to this is the most salient, the clinical: we would very much want to be able to offer the overall most efficient, safe systematic way to analyse a case, making it easy to do it right.

In summary, we are currently convinced to have good reason not to open for general sorting. Let's talk details and use cases though, and see if we can absolutely not cast into rank model terms? We have sometimes discussed a more explorative mode on the research side, when all variants for a case are expected to be loaded.
